frontend/src/components/organisms/form-sections/ÉquipeSection/Contact.tsx

```diff
@@ -175,6 +176,18 @@ export const Contact: React.FC<ContactProps> = ({
         required
       />
     )}
+    {type === 'demandeur' && !phone_number && (
+      <Alert type={AlertType.warning}>
+        Un numéro de téléphone est nécessaire : compléter sur{' '}
+        <a
+          href="https://moncomptepro.beta.gouv.fr/"
+          target="_blank"
+          rel="noreferrer"
+        >
+          Mon Compte Pro
+        </a>
+      </Alert>
```

Comment:
Can you maybe factorise this into an `<AlertMissingPhoneNumber />` component ? You can then use it in both `Contact.tsx` and `PersonnalInformationCard.tsx`

Reply:
Good idea !